felix/nftables/table.go

```diff
@@ -1075,9 +1073,7 @@ func (t *NftablesTable) applyUpdates() error {
 
 		newHashes[chainName] = newChainHashes
 		newChainToFullRules[chainName] = newRules
-
-		return nil // Delay clearing the set until we've programmed nftables.
```

Comment:
Losing a comment that is likely to be useful here.  Please reinstate the comment in the new version of the code,